### PR TITLE
Replace hero images

### PR DESCRIPTION
**Pull request by OlenaPotiatynyk**
## Description

Replace hero images on landing page

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/assets/image/hero1.png
Comment (mufniarz) [inline src/assets/image/hero1.png:1]: @Nyero2023 do we need to use this image, it's vary corporate office like.

Comment (Nyero2023) [reply src/assets/image/hero1.png:1]: > @Nyero2023 do we need to use this image, it's vary corporate office like.

Hey @mufniarz 
We can change the photo to something that goes well with a worker-owned cooperative.

I don't see what photo you are talking about here, the src/assets/image/hero1.png doesn't display on my end. But I believe you are talking about the first photo on the landing page right? (picture of an open space office)

Comment (mufniarz) [reply src/assets/image/hero1.png:1]: That's correct, everything you said :)

**Comment by mufniarz** [comment]
@OlenaPotiatynyk I think @Nyero2023 has a different image for the open space image. I'd replace that and fix the conflict PR and as long as it's passing we can merge this in.

**Pull request closed without merging**